### PR TITLE
Kristall: UI changes, view page source, new shortcuts, fixes for emojis and more

### PR DESCRIPTION
**Pull request by win8linux**
This updates Kristall to master as of commit https://github.com/MasterQ32/kristall/commit/ba7f39e9f0f329e78710c67e1267c1fbc79ac428. Some of the notable changes involve UI additions and tweaks, the addition of viewing page source, new shortcuts, and a fix for emojis that will hopefully be more permanent.

More detailed breakdown:

- [Add 'view source' to right-click context menu](https://github.com/MasterQ32/kristall/commit/237d3a8bc9ed6be7196115e0c7b640ca59349b1d)
- Shortcuts for [View page source](https://github.com/MasterQ32/kristall/commit/06b254d686a9fbed702e8d1777dc5c7adcba1fbe), [Settings](https://github.com/MasterQ32/kristall/commit/2ccc644f98019885562c9c5f46d27be39595ff06), [Quit](https://github.com/MasterQ32/kristall/commit/bf7e6a8a03f6a623b342d08c6e3202640aa40aba),  [Document Outline, Bookmark, and History docks](https://github.com/MasterQ32/kristall/commit/1ceae995f1984b2e9512752cf5e8d8b56d03d574)
- [Theme changes reflected instantly upon confirmation](https://github.com/MasterQ32/kristall/commit/9ee987f8216b412957ffcb46c01bbe268b282746)
- [Use system icon theme when using OS Default theme](https://github.com/MasterQ32/kristall/commit/ee1ad97ec70717f77e1e420fdfa2565ce3fa13c0)
- [URL bar styling](https://github.com/MasterQ32/kristall/commit/eabbbebed714f4d54f57fb9065ecbeb303f08f9b) and its respective [preference](https://github.com/MasterQ32/kristall/commit/40df924bc7289155a21a7c4ee901fc7c5e1673bd)
- [UI density option](https://github.com/MasterQ32/kristall/commit/19c8bf98885074617f9cc0a0b2f60ebf47ddf714)
- [Resolves the emoji problem, hopefully once and for all!](https://github.com/MasterQ32/kristall/commit/808288ce691a3c1520db513c7cff79fd22d3c53f)

### CODE REVIEW

File: www-client/kristall/kristall-0.7.3~git.recipe [hunk 1]
```diff
@@ -53,7 +53,6 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:qmake$secondaryArchSuffix >= 5
 	"
-
```
Comment (Begasus) [inline www-client/kristall/kristall-0.7.3~git.recipe:56]: Leave the new line between BUILD_PREREQUIRES and PATCH :)

**Comment by Begasus** [comment]
Watch the buildbot for eventual errors on the build :)

**Comment by Begasus** [comment]
For 32-bit: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/1032/builds/1754.log
For 64-bit: some packages already exist, revision bump required

**Comment by win8linux** [comment]
Whoops, missed that. I'll send a new PR bumping the rev soon.

**Pull request merged**